doc/source/api.rst

```diff
@@ -111,9 +111,7 @@ Google BigQuery
    read_gbq
    to_gbq
    generate_bq_schema
```

Comment:
sure. let's just deprecate it then (put a note in the gbq section) & remove from API.rst